api/src/main/java/jakarta/data/repository/package-info.java

```diff
@@ -16,25 +16,24 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 /**
- * <p>A repository is an interface annotated with {@link Repository} that defines
+ * <p>A repository is an interface annotated with {@link jakarta.data.repository.Repository} that defines
  * operations on entities. Entities represent data in the persistent store.
  * In Domain-driven design, a repository participates in the domain but abstracts away storage
  * and infrastructure details.</p>
  *
  * <p>Repository interfaces can optionally inherit from built-in interfaces within this package,</p>
  *
  * <ul>
- * <li>{@link DataRepository} - root of the hierarchy, allows the entity type to be specified as a type parameter.</li>
- * <li>{@link BasicRepository} - provides common find, delete, and save operations.</li>
- * <li>{@link CrudRepository} - extends the {@code BasicRepository} to add {@link CrudRepository#insert(S) insert}
- *     and {@link CrudRepository#update(T) update} operations.</li>
- * <li>{@link PageableRepository} - extends the {@code BasicRepository} with a built-in method that uses pagination.</li>
+ * <li>{@link jakarta.data.repository.DataRepository} - root of the hierarchy, allows the entity type to be specified as a type parameter.</li>
+ * <li>{@link jakarta.data.repository.BasicRepository} - provides common find, delete, and save operations.</li>
+ * <li>{@link jakarta.data.repository.CrudRepository} - extends the {@code BasicRepository} to add {@link jakarta.data.repository.CrudRepository#insert(java.lang.Object)}
+ *     and {@link jakarta.data.repository.CrudRepository#update(java.lang.Object)} operations.</li>
  * </ul>
  *
  * <p>Repository interfaces can also define their own life cycle methods using the
- * {@link Insert}, {@link Update}, {@link Save}, and {@link Delete} annotations,
+ * {@link jakarta.data.repository.Insert}, {@link jakarta.data.repository.Update}, {@link jakarta.data.repository.Save}, and {@link jakarta.data.repository.Delete} annotations,
  * as well as a variety of other methods following the Query by Method Name pattern,
- * the Parameter-based Conditions pattern, and the {@link Query} annotation.</p>
+ * the Parameter-based Conditions pattern, and the {@link jakarta.data.repository.Query} annotation.</p>
```

Comment:
It isn't wrong to add the package names here and above, but it should not be necessary given that these classes are in the same package as this package-info.java

Reply:
I put it, because it is broken at my IDE.

Do you mind to keep this way?

Reply:
Yes, it's fine.  We can look at how it gets generated later and make the assessment then if either way generates better.